e2e-tests/batch-mode-tests.sh

```diff
@@ -1,108 +1,90 @@
 done
-echo 'OPENMRS SERVER STARTED SUCCESSFULLY'
-​
-echo "BUILD THE ANALYTICS PROJECT"
-​
+echo "${print_prefix} OPENMRS SERVER STARTED SUCCESSFULLY"
+echo "${print_prefix} BUILD THE ANALYTICS PROJECT"
 mvn clean install
-​
-echo "RUN BATCH MODE WITH PARQUET SINK TESTS"
-
-java -cp batch/target/fhir-batch-etl-bundled-0.1.0-SNAPSHOT.jar org.openmrs.analytics.FhirEtl --openmrsServerUrl=http://localhost:8099/openmrs --openmrsUserName=admin --openmrsPassword=Admin123  --searchList=Patient,Encounter,Observation --batchSize=20 --targetParallelism=20 --fileParquetPath=/tmp/
-​
-if [ -z "$(ls /tmp/Patient)" ]; then
-  echo 'NO PARQUET FILES FOUND IN /tmp/'
+rm -rf /tmp/analytics-tests
+mktemp -d /tmp/analytics-tests
+echo "${print_prefix} RUN BATCH MODE WITH PARQUET SYNC"
+java -cp batch/target/fhir-batch-etl-bundled-0.1.0-SNAPSHOT.jar org.openmrs.analytics.FhirEtl --openmrsServerUrl=http://localhost:8099/openmrs --openmrsUserName=admin --openmrsPassword=Admin123  --searchList=Patient,Encounter,Observation --batchSize=20 --targetParallelism=20 --fileParquetPath=/tmp/analytics-tests/
+if [[ -z $(ls /tmp/analytics-tests/Patient) ]]; then
+  echo "${print_prefix} DATA NOT SYNCED TO PARQUET FILES"
   exit 0
 fi
-​
+echo "${print_prefix} PARQUET FILES SYNCED IN /tmp/analytics-tests DIRECTORY"​
 current_path=`pwd`
-
-echo "COPYING PARQUET TOOLS JAR FILE TO ROOT"
-
-cp parquet-tools-1.11.1.jar ~/
-​
-cd ~/
-
-echo "Finding number of patients, encounters and obs in openmrs server"
-​
-mkdir /tmp/omrs
-​
-curl -L -X GET -u admin:Admin123 http://localhost:8099/openmrs/ws/fhir2/R4/Patient/ >> /tmp/omrs/patients.json
-total_test_patients=$(jq '.total' /tmp/omrs/patients.json)
-echo "total_test_patients ---> $total_test_patients"
-​
-curl -L -X GET -u admin:Admin123 http://localhost:8099/openmrs/ws/fhir2/R4/Encounter/ >> /tmp/omrs/encounters.json
-total_test_encounters=$(jq '.total' /tmp/omrs/encounters.json)
-echo "total_test_encounters ---> $total_test_encounters"
-​
-curl -L -X GET -u admin:Admin123 http://localhost:8099/openmrs/ws/fhir2/R4/Observation/ >> /tmp/omrs/obs.json
-total_test_obs=$(jq '.total' /tmp/omrs/obs.json)
-echo "total_test_obs ---> $total_test_obs"
-​
-sudo rm -rf /tmp/omrs
-​
-echo "Counting number of patients, encounters and obs sinked to parquet files"
-​
-total_patients_streamed=$(java -jar ./parquet-tools-1.11.1.jar rowcount /tmp/Patient/)
-echo "total_patients_streamed ---> $total_patients_streamed"
-​
-total_encounters_streamed=$(java -jar ./parquet-tools-1.11.1.jar rowcount /tmp/Encounter/)
-echo "total_encounters_streamed ---> $total_encounters_streamed"
-​
-total_obs_streamed=$(java -jar ./parquet-tools-1.11.1.jar rowcount /tmp/Observation/)
-echo "total_obs_streamed ---> $total_obs_streamed"
-​
+echo "${print_prefix} COPYING PARQUET TOOLS JAR FILE TO ROOT"
+cp e2e-tests/parquet-tools-1.11.1.jar /tmp/analytics-tests/
+if ! command -v jq &> /dev/null; then #add !
```

Comment:
Can you please use `[[ ]]` for all `if`s?

Reply:
@bashir2 if I do that I get this error ```am getting Expected test to end here (don't wrap commands in []/[[]]). Fix any mentioned problems and try again```
I think this is supposed to be wrapped inside parenthesis?